integration-tests/utils/integration-utils.js

```diff
@@ -346,7 +347,11 @@ const withPageLogging = fn => async options => {
   return await fn(options)
 }
 
-const waitUntilLoadedOrTimeout = timeout => ({ waitUntil: ['load', 'domcontentloaded', 'networkidle0'], timeout })
+const navOptionNetworkIdle = (timeout = 60 * 1000) => ({ waitUntil: ['domcontentloaded'], timeout })
```

Comment:
The PR description says:
> Seeing sporadic UI test failures caused by Navigation timeout.
> ...
> I think it has to do with waitUntil: load, domcontentloaded options so I'm going to remove these two.

But this removes `load` and `networkidle0`, not `domcontentloaded`. What changed?

Also, the function name is misleading since `navOptionNetworkIdle` does not wait for `networkidle`.